app/src/main/java/com/kickstarter/ui/activities/ProfileActivity.kt

```diff
@@ -2,126 +2,138 @@ package com.kickstarter.ui.activities
 
 import android.content.Intent
 import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.viewModels
 import androidx.core.view.isGone
 import androidx.recyclerview.widget.GridLayoutManager
 import androidx.recyclerview.widget.LinearLayoutManager
 import com.kickstarter.R
 import com.kickstarter.databinding.ProfileLayoutBinding
-import com.kickstarter.libs.BaseActivity
-import com.kickstarter.libs.RecyclerViewPaginator
-import com.kickstarter.libs.qualifiers.RequiresActivityViewModel
-import com.kickstarter.libs.rx.transformers.Transformers.observeForUI
+import com.kickstarter.libs.recyclerviewpagination.RecyclerViewPaginatorV2
+import com.kickstarter.libs.rx.transformers.Transformers.observeForUIV2
 import com.kickstarter.libs.utils.ApplicationUtils
 import com.kickstarter.libs.utils.ViewUtils
+import com.kickstarter.libs.utils.extensions.addToDisposable
+import com.kickstarter.libs.utils.extensions.getEnvironment
 import com.kickstarter.libs.utils.extensions.getProjectIntent
 import com.kickstarter.models.Project
 import com.kickstarter.ui.IntentKey
 import com.kickstarter.ui.adapters.ProfileAdapter
 import com.kickstarter.ui.extensions.loadCircleImage
+import com.kickstarter.ui.extensions.startActivityWithTransition
 import com.kickstarter.viewmodels.ProfileViewModel
+import io.reactivex.disposables.CompositeDisposable
 
-@RequiresActivityViewModel(ProfileViewModel.ViewModel::class)
-class ProfileActivity : BaseActivity<ProfileViewModel.ViewModel>() {
+class ProfileActivity : ComponentActivity() {
```

Comment:
are we clearing the disposables anywhere here?

Reply:
good catch, forgot about that